lib/client/api.go

```diff
@@ -2786,7 +2786,18 @@ func Username() (string, error) {
 	if err != nil {
 		return "", trace.Wrap(err)
 	}
-	return u.Username, nil
+
+	username := u.Username
+
+	// If on Windows, strip the domain name.
+	if runtime.GOOS == "windows" {
+		idx := strings.LastIndex(username, "\\")
+		if idx > -1 {
```

Comment:
I would check the upper bound too - unlikely but we don't want to panic with out of bounds error if the `\` is the last character.

```suggestion
		if idx > -1 && index < len(username) -1 {
```